src/main/java/org/reactome/addlinks/referencecreators/SimpleReferenceCreator.java

```diff
@@ -20,7 +16,7 @@
 
 /**
  * Creates references from one database to another.
- * @param T - The type that will be mapped to in the createIdentifiers method. <i>This</i> class will assume String,
+ * @param <T></T> - The type that will be mapped to in the createIdentifiers method. <i>This</i> class will assume String,
```

Comment:
I think `<T></T>` should probably just be `T` - it looks like the IDE's Javadoc autocorrect got a little carried away ? ;)

Reply:
You hit that nail on the head